server/config/database.js

```diff
@@ -1,5 +1,8 @@
 export default {
   get url () {
+
+    process.env.NODE_ENV=process.env.NODE_ENV|| "dev";
```

Comment:
Instead of overwriting `process.env.NODE_ENV`, try to use the switch `default` case instead.